README.md

```diff
@@ -40,40 +40,150 @@ After this, a record for ID `doms.radio:albert-einstein.xml` should be delivered
 
-After this Solr is available at http://localhost:10007/solr/ and can be stopped and started with
-```
+After this Solr is available at http://localhost:10007/solr/ 
+
+### Operating Solr
+
+The cloud can be stopped and started with
+```shell
 bin/cloud_stop.sh
 bin/cloud_start.sh
 ```
 
-If the Solr configuration is changed, force an update of `ds` with
+Aliases and collections can be seen from the admin gui or by calling
+```shell
+bin/cloud_alias.sh
+```
+and
+```shell
+bin/cloud_status.sh
+```
+
+Clear all records in the `ds-20231114-1446` collection, but keep the collection:
+```shell
+bin/cloud_clear.sh ds-20231114-1446
+```
+
+Fully delete a concrete collection, e.g. `ds-20231114-1446`:
+```shell
+bin/cloud_delete.sh ds-20231114-1446
+```
+
+### Configuration update without reindex
+
+
+If the Solr configuration is changed in a way that does not require a full reindex, the
+configuration can be assigned to an existing collection.
+
+Start by determining what the current collection is by calling
+```shell
+bin/cloud_alias.sh
+```
+This should give an output such as 
 ```
-FORCE_CONFIG=true bin/cloud_sync.sh src/main/solr/dssolr/conf/ ds-conf ds
+Aliases:
+{
+  "ds-write": "ds-20231116-1045",
+  "ds": "ds-20231116-1045"
+}
+
+Collections:
+ds-20231114-1312
+ds-20231116-1045
 ```
 
-Clear all records in the `ds` collection, but keep the collection with
+Extract the current version and update the configuration for the collection with the following (note that the collection name must be entered explicitly. In this case it's `ds-20231116-1045`):
+
+```shell
+CONF_VERSION=$(grep 'name="_ds_.*_"' src/main/solr/dssolr/conf/schema.xml  | sed 's/.*name="_ds_\([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\)_.*/\1.\2.\3/') ; echo "ds-conf version: $CONF_VERSION"
+bin/cloud_sync.sh src/main/solr/dssolr/conf/ ds-conf-${CONF_VERSION} ds-20231116-1045
 ```
-bin/cloud_clear.sh ds
+
+If this is during development and the Solr config version has not been bumped, the update can be forced:
+```shell
+CONF_VERSION=$(grep 'name="_ds_.*_"' src/main/solr/dssolr/conf/schema.xml  | sed 's/.*name="_ds_\([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\)_.*/\1.\2.\3/') ; echo "ds-conf version: $CONF_VERSION"
+FORCE_CONFIG=true bin/cloud_sync.sh src/main/solr/dssolr/conf/ ds-conf-${CONF_VERSION} ds-20231116-1045
 ```
 
-Fully delete the `ds` collection with
+
+### Full reindex
+
+If a fresh index is needed, the alias mehanism ensures that this is done without disrupting
+the existing collection.
+
+Create a new empty index with
+```shell
+  COLLECTION="ds-$(date +%Y%m%d-%H%M)" ; echo "Collection: $COLLECTION"
+  CONF_VERSION=$(grep 'name="_ds_.*_"' src/main/solr/dssolr/conf/schema.xml  | sed 's/.*name="_ds_\([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\)_.*/\1.\2.\3/') ; echo "ds-conf version: $CONF_VERSION"
+  bin/cloud_sync.sh src/main/solr/dssolr/conf/ ds-conf-${CONF_VERSION} $COLLECTION
+  bin/cloud_alias.sh ds-write $COLLECTION
```

Comment:
Again, could this be made prettier as a tiny script in itself?

Reply:
As above so below.